python/lsst/ts/hvac/simulator/sim_client.py

```diff
@@ -119,18 +119,67 @@ def publish_mqtt_message(self, topic, payload):
         ------
         ValueError
             In case a topic doesn't exist.
-        ValueError
-            In case a different command than COMANDO_ENCENDIDO_LSST is
-            received.
         """
         self.log.debug(f"Recevied message on topic {topic} with payload {payload}")
         topic, command = xml.extract_topic_and_item(topic)
-        if command != "COMANDO_ENCENDIDO_LSST":
-            raise ValueError(f"Command {command} not supported on topic {topic}")
-        value = json.loads(payload)
-        self.topics_enabled[topic] = value
+        if command == "COMANDO_ENCENDIDO_LSST":
+            self._handle_enable_command(topic, json.loads(payload))
+        else:
+            self._handle_config_command(topic, command, json.loads(payload))
+        # For now, always return True
```

Comment:
Thanks. I expanded the comment with `It is unclear if the real MQTT server ever returns False and once that is clear this will be adapted` and rewrote the method doc string.